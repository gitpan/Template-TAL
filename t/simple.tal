<template>
  text here
  <br/>
</template>
