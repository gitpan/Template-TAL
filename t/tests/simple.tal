<html xmlns:tal="http://xml.zope.org/namespaces/tal">
  <body>
    <h1 tal:content="title">The title</h1>
    <ul>
      <li tal:repeat="colour colours" tal:content="colour">Colour of the rainbow</li>
    </ul>
  </body>
</html>
