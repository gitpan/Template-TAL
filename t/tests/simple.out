<html>
  <body>
    <h1>some title</h1>
    <ul>
      <li>red</li>
      <li>green</li>
      <li>blue</li>
    </ul>
  </body>
</html>
